serde/src/java/org/apache/hadoop/hive/serde2/JsonSerDe.java

```diff
@@ -142,227 +197,21 @@ public Object deserialize(Writable blob) throws SerDeException {
    * and generate a Text representation of the object.
    */
   @Override
-  public Writable serialize(Object obj, ObjectInspector objInspector)
-    throws SerDeException {
-    StringBuilder sb = new StringBuilder();
-    try {
+  public Writable serialize(final Object obj,
```

Comment:
is this split on 2 lines for reason or it is github rendering ?

Reply:
I just checked this... this is the correct format when using the Hadoop check-style format.